app/client/packages/design-system/ads/src/DismissibleTab/DismissibleTabBar.types.ts

```diff
@@ -4,7 +4,11 @@ import type { DismissibleTabProps } from "./DismissibleTab.types";
 export interface DismissibleTabBarProps {
   children:
     | React.ReactElement<DismissibleTabProps>
-    | React.ReactElement<DismissibleTabProps>[];
-  onTabAdd: () => void;
+    | React.ReactElement<DismissibleTabProps>[]
+    | React.ReactNode;
+  className?: string;
   disableAdd?: boolean;
+  hideAdd?: boolean;
+  isAddingNewTab?: boolean;
```

Comment:
This was confusing to understand. Can we add a comment in to explain

```suggestion
  /* Shows a loading indicator in place of add button  */
  isAddingNewTab?: boolean;
```